src/model/extension.ts

```diff
@@ -135,6 +136,45 @@ export class DataFilterExtension extends BaseExtensionModel {
   }
 }
 
+export class PathStyleExtension extends BaseExtensionModel {
+  static extensionType = "path-style";
+
+  extensionInstance: _PathStyleExtension;
+
+  constructor(
+    model: WidgetModel,
+    layerModel: BaseLayerModel,
+    updateStateCallback: () => void,
+  ) {
+    super(model, updateStateCallback);
+
+    const dash = this.model.get("dash");
+    const highPrecisionDash = this.model.get("high_precision_dash");
+    const offset = this.model.get("offset");
+    this.extensionInstance = new _PathStyleExtension({ dash });
+
+    // Properties added by the extension onto the layer
+    layerModel.initRegularAttribute("get_dash_array", "getDashArray");
```

Comment:
Anything that starts with `get` is an "accessor" not an "attribute", so this should be `initVectorizedAccessor` instead. (If you could group the `initVectorizedAccessor` lines at the bottom that would be great)

Reply:
And by "accessor" I mean that the user can pass in either a scalar to apply to _every_ row or a column of values where each row applies to the geometry at the same index.